packages/custom-esbuild/README.md

```diff
@@ -129,6 +135,25 @@ const defineTextPlugin: Plugin = {
 export default defineTextPlugin;
 ```
 
+OR:
+
+```ts
+// esbuild/plugins.ts
+import type { Plugin, PluginBuild } from 'esbuild';
+
+function defineRewritePathPlugin(options: { text: string }): Plugin {
```

Comment:
This should be the `esbuild/define-env.ts` example, along with a practical illustration of how the `stage` parameter could be utilized. Please avoid providing text examples, as they are unnecessary

Reply:
is updated